src/templates/ensembles.js

```diff
@@ -30,9 +29,12 @@ export const EnsemblesTemplate = ({
     <div className="ensembles__content">
       <Header topText={topText} bottomText={bottomText} />
       <TextContent header={subheading} content={about} />
-      <Button link={buttonLink}>
-        {buttonText}
-      </Button>
+      <FloatingCard
```

Comment:
![Screen Shot 2020-04-04 at 10 08 12 PM](https://user-images.githubusercontent.com/14127129/78465256-26cdab00-76c1-11ea-92c9-d0bd214f3d1e.png)
The space at the bottom seems a little much without the email icon